### DISCUSSION TITLE
Any way to prevent cmd-tab from switching workspaces?

### DISCUSSION THREAD

**Discussion by ferdinandrau** [nikitabobko/AeroSpace#812] [questions-and-answers] [answered]
AeroSpace is awesome, the one thing that annoys me when using it though is that doing cmd+tab to switch between running applications switches to a space with one of the app's windows. I would like to never switch spaces unless explicitly requested, e.g. with ctrl+4 to go to workspace 4, while still using cmd+tab to switch the active app to be able to spawn new windows with cmd+n on the current space. Is that possible? It was in yabai, but obviously this tool does not actually use real spaces, so I'm not sure...

**Comment by nikitabobko** [comment, marked as answer]
It's technically not possible, since AeroSpace is not focus owner https://github.com/nikitabobko/AeroSpace/issues/571#issuecomment-2400968083
